src/WSRequestHandler_Outputs.cpp

```diff
@@ -129,6 +129,8 @@ RpcResponse WSRequestHandler::GetOutputInfo(const RpcRequest& request)
 /**
 * Start an output
 *
+* Note: Controlling outputs is not a heavily tested feature of obs-websocket. Some plugins which add outputs to OBS may not function properly when they are controlled in this way.
```

Comment:
We should call that "experimental feature".

```suggestion
* Note: Controlling outputs is an experimental feature of obs-websocket. Some plugins which add outputs to OBS may not function properly when they are controlled in this way.
```